platform/landing-page/src/main/resources/OSGI-INF/blueprint/blueprint.xml

```diff
@@ -20,7 +20,7 @@
         <property name="path" value=""/>
     </bean>
     <service id="resources" ref="resourceMapping"
-             interface="org.ops4j.pax.web.extender.whiteboard.ResourceMapping"/>
+             interface="org.ops4j.pax.web.service.whiteboard.ResourceMapping"/>
```

Comment:
Good catch. It looks like the package name for the `DefaultResourceMapping` class still uses `extender` instead of `service`: https://github.com/ops4j/org.ops4j.pax.web/blob/web-6.0.3/pax-web-extender-whiteboard/src/main/java/org/ops4j/pax/web/extender/whiteboard/runtime/DefaultResourceMapping.java.